docs/12-design-changes.md

```diff
@@ -0,0 +1,24 @@
+# Design Changes
+
+###### WIP Change: Improve "Offers"
+* Discussion on the next conference: Dissolve Extras category into Jobs, Apartment, Other (Other as category for extra wishes) → There is no icon and no wording that summarizes all extras, through the categorized presentation, the offers become more visible and more newcomers can benefit
```

Comment:
```suggestion
* Discussion on the next conference: Break Extras up into Jobs, Apartment, Other (Other as category for extra wishes) → There is no icon and no wording that summarizes all extras, through the categorized presentation, the offers become more visible and more newcomers can benefit
```

> `Other` as category for extra wishes

Wishes of a certain municipality?

Reply:
Result of the design workshop. Discussio will follow.